components/sanckbar.css

```diff
@@ -0,0 +1,10 @@
+@import url(/stylesheets/main.css);
+
+.snackbar-container{
+    width: 100%;
+    background-color: hsl(var(--primary-lighter));
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
You can extract this in a utility class so that it can be used elsewhere as well